docs/fundamentals/flow/flow-api.md

```diff
@@ -211,10 +211,46 @@ f = Flow(protocol='http', no_debug_endpoints=True, no_crud_endpoints=True)
 
 After setting up a Flow in this way, the {ref}`default HTTP endpoints <custom-http>` will not be exposed.
 
-### Add GraphQL endpoint
+(cors)=
+### Enable Cross-Origin Resource Sharing (CORS)
 
-````{admonition} Attention
-:class: attention
+To make a Flow accessible from a website with a different domain, you need to enable [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
+Among other things, CORS is necessary to provide a {ref}`Swagger UI interface <swagger-ui>` for your Flow.
+
+Not that CORS is disabled by default, for security reasons.
+To enable CORS, configure your Flow in the following way:
+```python
+from jina import Flow
+
+f = Flow(cors=True, protocol='http')
+```
+
+### Advanced configuration options
+
+HTTP support in Jina is powered by [Uvicorn](https://www.uvicorn.org/).
+You can configure the Flow's internal Uvicorn sever to your heart's content by passing `uvicorn_kwargs` to the Flow:
```

Comment:
your heart's content?

Reply:
Yeah, as much as you would like. Can use less flowery language if this is too much